### DISCUSSION TITLE
How to enable fp8?

### DISCUSSION THREAD

**Discussion by giteeeeee** [lllyasviel/stable-diffusion-webui-forge#520] [Q&A]
![1232024-03-09 191607](https://github.com/lllyasviel/stable-diffusion-webui-forge/assets/89521447/6b8785bd-9049-4fa5-951b-b4fa55f73ad0)

Activating this option hasn't been working for me. VRAM and the image generated stay the same.

**Comment by leomaxwell973** [comment]
Just out of curiosity, did you test while using a SDXL-lora on an SDXL model while testing? iirc it interfaces with loras quite directly in SDXL mode, or something like that. but who knows, had a script for a button missing some lines for whatever reason and half the program shuts down so, be sure to be sure i guess xD

**Comment by fm3at** [comment]
use --unet-in-fp8-e4m3fn launch argument

**Comment by Edu2014** [comment]
![image](https://github.com/user-attachments/assets/dc6cde43-a1a2-4653-a3e4-4516660ca40d)


It's not showing up anymore, how do I get it back, it's not in the menu either.
